x-pack/plugins/enterprise_search/public/applications/app_search/components/credentials/credentials_list/key.tsx

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+import { EuiButtonIcon } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+
+interface Props {
```

Comment:
just wondering (I have no strong feelings whatsoever myself), do we still want to keep doing the `ISomething` prefix for interfaces (so in this case `IProps`?) I think we used to lint on it in the ent-search repo but we don't in Kibana in seems. Again, I'm fine with whatever, just curious on what we think as a team

Reply:
Good catch, I think this should be IProps.

Reply:
acec242a135a1c17e289de04b27dfbf60a44d015